15-lhcb-dev.md

```diff
@@ -27,20 +27,20 @@ Successfully created the local project DaVinciDev in .
 To start working:
 
   > cd ./DaVinciDev
-  > getpack MyPackage vXrY
+  > git lb-use DaVinci
+  > git lb-checkout DaVinci/vXrY MyPackage
 
 then
 
   > make
   > make test
-  > make QMTestSummary
```

Comment:
OK, I guess the actual `lb-dev` script itself has been updated (along with the default Makefile that's generated).